tasks/travis-pr-title-lint.js

```diff
@@ -152,12 +154,30 @@ module.exports = function(grunt) {
         return;
       }
 
+      var config = path.join(__dirname, '..', '.conventional-changelog-lintrc');
+      var checkLintConfig = Promise.all([
+        fs.stat(config), fs.access(config, fs.R_OK),
+      ])
+        .then(function(results) {
+          const configStat = results[0];
+
+          if (!configStat.isFile()) {
+            throw new Error(
+              `Config file "${config}" found, but it should be a regular file`
+            );
+          }
+        })
+        .catch(function(err) {
+          throw new Error(`Unable to load changelog lint config: ${err.stack}`);
+        });
+
       Promise.all([
+        checkLintConfig,
         getPreset('angular'),
-        getConfiguration('.conventional-changelog-lintrc'),
+        getConfiguration(),
```

Comment:
yep, unfortunately the getConfiguration helper doesn't take the filename, but a series of parameter that will affect the final config file name :-)

Code References:
- https://github.com/marionebl/conventional-changelog-lint/blob/875a77d66cf09f8106fc348e85b8a6ddcbcfa55d/source/cli.js#L103-L109
- https://github.com/marionebl/conventional-changelog-lint/blob/master/source/library/get-configuration.js#L11-L17